File: app/src/main/java/com/example/alkewalletandroid/MainActivity9.java
package com.example.alkewalletandroid;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity9 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main9);

        ImageView volver = findViewById(R.id.perfil_cuenta);

        volver.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity9.this, MainActivity5.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/alkewalletandroid/view/MainActivity9.kt
package com.example.alkewalletandroid.view

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity
import com.example.alkewalletandroid.R

class MainActivity9 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main9)
        val volver = findViewById<ImageView>(R.id.perfil_cuenta)
        volver.setOnClickListener {
            val intent = Intent(this@MainActivity9, MainActivity5::class.java)
            startActivity(intent)
        }
    }
}